### DISCUSSION TITLE
Confused when using `border` and `outline` utility

### DISCUSSION THREAD

**Discussion by milhamm** [tailwindlabs/tailwindcss#12730] [General]
I was confused when I used both classes. I was expecting `outline` utility class to output `outline-width: 1px` similar to `border`. However, `outline` is used for `outline-style: solid`. 

I have no issue with this tho 😄 . Just out of curiosity, is it by design or there will be any changes to make both of them similar?  

Thanks!

**Comment by obecker** [comment]
I was just about to create the same question. In my eyes these classes should be aligned, i.e. 
  - `outline`  → `outline-solid`
  - `outline-1` → `outline`

Maybe something for `v4`?
